generic/tests/cfg/iometer_windows.cfg

```diff
@@ -73,3 +73,35 @@
                     bg_mode = yes
                 - after_test:
                     bg_mode = no
+        - with_multi_scsi_disks:
+            only virtio_scsi
+            run_timeout = 3600
+            format_multi_disks = yes
+            images = "image1"
```

Comment:
could this line can be deleted? it seems no need add here, the default system image is image1~

Reply:
that's ok~